### ISSUE TITLE
registry files are not working on some systems

### ISSUE THREAD

**Issue by FunkyFr3sh** [solemnwarning/ipxwrapper#23]
On a german system the registry path is not `IPX Connection For DirectPlay` , it is `IPX-Verbindung für DirectDraw` instead

![image](https://github.com/user-attachments/assets/1de7ce2d-754c-4bd4-842c-5207492ced04)

**Comment by solemnwarning** [comment]
> Hey, lets localise bits of paths in the registry, wouldn't that be funny?

- Someone at Microsoft, probably

Thanks for reporting this, I guess I'll replace the reg files with a utility that checks for an existing provider by its Guid before updating it or registering a new one.

**Comment by FunkyFr3sh** [comment]
Yep, it's really weird.

Done some more testing:

On a new german windows 7 install I do not have the german reg keys, only the english ones.

What's weird, if I rename the key to "IPX-Verbindung für DirectPlay1" (appended 1 at the end) then it's still broken. BUT, if I prefix it with a 1 like "1IPX-Verbindung für DirectPlay" then it would actually switch to the english key name and it works. This is a really a mess. 

So it seems to read both, but the order matters. Heres how it looks like with the 1 prefix (working)
![image](https://github.com/user-attachments/assets/af2df182-36d0-4fe8-afc1-f278c9697f60)

Here original (broken)
![image](https://github.com/user-attachments/assets/f6aec142-c9fb-4b33-88fc-f6a4b8e4876a)

**Comment by solemnwarning** [comment]
Yep, the GUID is used as the key for this. Games which hard-code a specific service provider use that.

**Comment by solemnwarning** [comment]
[IPXWrapper 0.7.2](https://github.com/solemnwarning/ipxwrapper/releases/tag/ipxwrapper-0.7.2) released.
